contact.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Sprint Challenge - Home</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+</head>
+
+
+    <body>
+        
+    <div class="container">
+            
+        <header>
+        
+                <nav>
+                    <img src="img/lambda-black.png" alt="">
+                    
+                    <a href="index.html">Home</a>
+                    <a href="about.html">About</a>
+                    <a href="#">Products</a>
+                    <a href="#">Blog</a>
+                    <a href="#">Contract</a>
+                    </nav>
+                        
+        </header>
+
+    <img id="contact-image" src="img/Contact.png">
+    
+        <div class="contact-container">
+
+                <form action="/action_page.php">
```

Comment:
Awesome job researching and building out this contact form! We will be going over this later in the track, so you're a step ahead.

Reply:
Thanks, I spent a long time researching it for the UI project!